app/controllers/objects_controller.rb

```diff
@@ -9,7 +9,10 @@ class ObjectsController < ApplicationController
   # return the PreservedObject model for the druid (supplied *without* `druid:` prefix)
   # GET /v1/objects/:id.json
   def show
-    render json: PreservedObject.find_by!(druid: druid).to_json
+    obj = JSON.parse(PreservedObject.find_by!(druid: druid).to_json)
```

Comment:
How would you feel about moving the logic in these lines into the `PreservedObject` model? So, something like:

```ruby
  def as_json(*)
    super.except('id', 'preservation_policy_id')
  end
```